addons/storyshots/src/index.js

```diff
@@ -29,36 +27,40 @@ export default function testStorySnapshots(options = {}) {
     options.framework === 'react-native' || hasDependency('@storybook/react-native');
 
   if (isStorybook) {
-    storybook = require.requireActual('@storybook/react');
+    getStorybook = require.requireActual('@storybook/react').getStorybook;
     // eslint-disable-next-line
-    const loadBabelConfig = require('@storybook/react/dist/server/babel_config')
-      .default;
+    // const loadBabelConfig = require('@storybook/react/dist/server/babel_config').default;
     const configDirPath = path.resolve(options.configPath || '.storybook');
     configPath = path.join(configDirPath, 'config.js');
 
-    const babelConfig = loadBabelConfig(configDirPath);
-    const content = babel.transformFileSync(configPath, babelConfig).code;
-    const contextOpts = {
-      filename: configPath,
-      dirname: configDirPath,
-    };
+    // const babelConfig = loadBabelConfig(configDirPath);
```

Comment:
Well correct me if I'm wrong: jest can do babel transpilation, why are we doing it?
I don't understand why we're using require.requireActual either